src/org/openstreetmap/josm/plugins/pt_assistant/gui/PTAssistantPaintVisitor.java

```diff
@@ -45,8 +42,9 @@
 public class PTAssistantPaintVisitor extends PaintVisitor {
 
 	private static final String Node = null;
-	private final static Set<String> PUBLIC_TRANSPORT_NODE_ROLES = new HashSet<>(Arrays.asList("stop",
-			"stop_entry_only", "stop_exit_only", "platform", "platform_entry_only", "platform_exit_only"));
+	private final static String[] PUBLIC_TRANSPORT_NODE_ROLES = {
+		"stop", "stop_entry_only", "stop_exit_only", "platform", "platform_entry_only", "platform_exit_only"
+	};
```

Comment:
This variable was unused but there were occurences of the variable name in a string, so now the variable is used instead of that string.